tests/unit/engine.rs

```diff
@@ -3,6 +3,8 @@ mod tests {
     use super::super::*;
     use crate::lists::FilterFormat;
     use crate::resources::MimeType;
+    use base64::prelude::BASE64_STANDARD;
+    use base64::engine::Engine as _;
```

Comment:
could combine these like `use base64::{prelude::BASE64_STANDARD, engine::Engine as _};`

and I think it'd be good replace all the `prelude::*` imports this way to avoid similar confusion in the future